### ISSUE TITLE
[openbmc] rflash does not understand relative path for .tar file

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3670]
Passing a relative path for the .tar file does not work...

```
[root@stratton01 ob910.1732.20170807n]# pwd
/root/vhu/ob910.1732.20170807n
[root@stratton01 ob910.1732.20170807n]# rflash frame6u05 -u ./witherspoon_910_golden.pnor.squashfs.tar
Error: Cannot access ./witherspoon_910_golden.pnor.squashfs.tar
```

But passing in full path does: 
```
[root@stratton01 ob910.1732.20170807n]# rflash frame6u05 -u /root/vhu/ob910.1732.20170807n/witherspoon_910_golden.pnor.squashfs.tar
frame6u05: [openbmc_debug] POST https: //10.6.5.254/login -d { "data": [ "root", "0penBmc" ] }
frame6u05: [openbmc_debug] login_response 200 OK
frame6u05: [openbmc_debug] PUT https: //10.6.5.254/xyz/openbmc_project/upload/image/ -T /root/vhu/ob910.1732.20170807n/witherspoon_910_golden.pnor.squashfs.tar
frame6u05: [openbmc_debug] rflash_file_upload_response 405 Method Not Allowed
frame6u05: Update file /root/vhu/ob910.1732.20170807n/witherspoon_910_golden.pnor.squashfs.tar successfully uploaded
```

The file is in fact uploaded...  but with debug turned on we see that "405 Method Not Allowed"  Is this normal?  `frame6u05: [openbmc_debug] rflash_file_upload_response 405 Method Not Allowed`

**Comment by whowutwut** [comment]
Suggest change the message to something like:  
```
[root@stratton01 ob910.1732.20170807n]# rflash frame6u05 -u /root/vhu/ob910.1732.20170807n/witherspoon-redbud.20170808t.pnor.squashfs.tar
frame6u05: Uploading /root/vhu/ob910.1732.20170807n/witherspoon-redbud.20170808t.pnor.squashfs.tar... 
frame6u05: Successful, use the -l option to list. 
```

**Comment by gurevichmark** [comment]
Implemented by #3684

**Comment by whowutwut** [comment]
Verified this yesterday when flashing on Bostons...
